Makefile

```diff
@@ -81,6 +81,10 @@ generate-cli-reference:
 prepare-release: cross
 	./scripts/prepare-release.sh
 
+.PHONY: configure-installer-tests-cluster
+configure-installer-tests-cluster:
+	. ./scripts/configure-installer-tests-cluster.sh
```

Comment:
please make that script executable, so it is aligned with the rest of the scripts in `scripts`, you can also get rid of the firs dot

Reply:
I can add executable. On the contrary it will be needed or export of KUBECONFIG will end once script execution completes. The script runs a child of current and hence any envs exported will no longer be valid in parent